--- ./extra/Python/setup.py
+++ ./extra/Python/setup.py
@@ -29,7 +29,7 @@
        ext_modules = [Extension("lpsolve55",
 				["lpsolve.c", "hash.c", "pythonmod.c"],
                                 define_macros=[('PYTHON', '1'), (WIN32, '1'), ('NODEBUG', '1'), ('DINLINE', 'static'), (NUMPY, '1'), ('_CRT_SECURE_NO_WARNINGS', '1')],
-                                include_dirs=['../..', NUMPYPATH],
+                                include_dirs=[p + '/Library/include/lpsolve', NUMPYPATH],
                                 library_dirs=[LPSOLVE55],
 				libraries = ["lpsolve55"])
 		      ]
